src/constants/locale.ts

```diff
@@ -1,4 +1,4 @@
-export const SUPPORTED_LOCALES = ['en', 'zh', 'ru', 'tr']
+export const SUPPORTED_LOCALES = ['en', 'zh', 'ru', 'tr', 'pt']
```

Comment:
@johnnyd-eth looks like `pt` is commented out elsewhere and that this should be adding `es` instead of `pt`, but i see the branch is called add-portuguese lol. so i'm not sure if i know what's going on here

Reply:
Oh damn my bad! Yeah I originally thought it was the Portuguese translations that were ready but it was actually the Spanish, so had to re-update these configurations to es